packages/calcite-components/src/components/color-picker/resources.ts

```diff
@@ -93,4 +93,5 @@ export const DIMENSIONS = {
       radius: 10,
     },
   },
+  thumbNode: 1,
```

Comment:
Can this be moved to the component itself or a separate constant? Currently, I find it a bit confusing:

1. the dimensions object stores the size of different parts across all scales that are drawn onto canvas elements
2. I know it's a bit verbose, but repeated values across scales have a corresponding prop
3. none of the existing part names use `node`, and there's already one for thumb